windows/src/engine/keyman32/capsstate.cpp

```diff
@@ -63,15 +57,17 @@ void KeyCapsLockPress(BOOL FIsUp)  // I3284 - void   // I3529
 
 	if(_td->lpActiveKeyboard->Keyboard->dwFlags & KF_CAPSONONLY)
 	{
-		if(FIsUp && !(GetKeyState(VK_CAPITAL) & 1))		// I267 - 24/11/2006 invert GetKeyState test
+    SendDebugMessageFormat(0, sdmAIDefault, 0, "KeyCapsLockPress: KF_CAPSONONLY: FIsUp=%d CapsState=%d", FIsUp, GetKeyState(VK_CAPITAL) & 1);
+    if(FIsUp && !(GetKeyState(VK_CAPITAL) & 1))		// I267 - 24/11/2006 invert GetKeyState test
```

Comment:
1. It seems there's a lot of `GetKeyState(VK_CAPITAL) & 1` in this function. Is it possible to just call it once above the `if()` block?

2. tabspace is making the indents look off.

Reply:
1. I opted to make a function `IsCapsLockOn()` rather than introduce locals to cache what is a very quick call.

2. Fixed tab stops on this file